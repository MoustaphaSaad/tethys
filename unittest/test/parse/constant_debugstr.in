constant msg "Hello, World!\0"

proc main
	u64.load r1 msg
	debugstr r1
	halt
end