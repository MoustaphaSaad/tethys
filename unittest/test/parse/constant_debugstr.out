constant msg "Hello, World!\0"
PROC main
  u64.load r1 msg
  INVALID OP
  halt
END
